odoo/addons/base/models/ir_module.py

```diff
@@ -393,6 +393,8 @@ def _state_update(self, newstate, states_to_update, level=100):
     @assert_log_admin_access
     def button_install(self):
         company_countries = self.env['res.company'].search([]).country_id
+        if self.country_ids:
+            company_countries += self.country_ids
```

Comment:
You could write it as
```suggestion
        company_countries |= self.country_ids
```
but I don't think that's correct anyway, below the `must_install` contains `module.country_ids & company_countries` which will always be true after your patch. Needs to be adapted.

Reply:
No, `module` does not have the same value as `self`

Reply:
`module` has value with search with `auto_domain` and self doesn't so it's not always true